src/payout_queue/repo.rs

```diff
@@ -4,158 +4,168 @@ use tracing::instrument;
 use std::collections::HashMap;
 
 use super::{entity::*, error::PayoutQueueError};
-use crate::{entity::*, primitives::*};
+use crate::{primitives::*};
+use es_entity::*;
 
-#[derive(Debug, Clone)]
-pub struct PayoutQueues {
+#[derive(EsRepo, Clone, Debug)]
+#[es_repo(
+    entity = "PayoutQueue",
+    err = "PayoutQueueError",
+    columns(id(ty = "PayoutQueueId", list_for, update(persist = false))),
+    tbl_prefix = "core"
```

Comment:
`core` prefix is not required for this.. 
If you add the `core` prefix it will look for table called `core_payout_queues` something like that